tests/features/eulogin/eulogin.feature

```diff
@@ -327,11 +328,6 @@ Feature: Log in through EU Login
     Then I should see the error message "First name field is required."
     And I should see the error message "Family name field is required."
 
-  Scenario: The Drupal login form shows a warning message.
-    When I visit "/user/login"
-    Then I should see the warning message "As of 01/02/2020, EU Login will be the only authentication method available on Joinup. So, we strongly recommend you to choose EU Login as your preferred sign-in method!"
-    And I should see the link "EU Login"
-
```

Comment:
This should not be deleted but replaced with the new warnung text, provided by Filip.